src/RecoveryServices/RecoveryServices.Backup.Models/CommonModels/PolicyRetentionObjects.cs

```diff
@@ -229,9 +229,9 @@ public class DailyRetentionSchedule : RetentionScheduleBase
         // no extra fields
         public override void Validate()
         {
-            if (DurationCountInDays <= 0 || DurationCountInDays > PolicyConstants.MaxAllowedRetentionDurationCount)
+            if (DurationCountInDays < 7 || DurationCountInDays > PolicyConstants.MaxAllowedRetentionDurationCount)
```

Comment:
@siddharth7 how will this affect existing scripts that use one to six days? Will they be broken?

Reply:
Existing scripts will throw an error while creating or modifying a policy. This is a service side change that has been brought in and we are changing the client side validation accordingly.